dali/pipeline/operators/reader/coco_reader_op.cc

```diff
@@ -309,6 +309,13 @@ Tensor (`m` * `[x, y, w, h] or `m` * [left, top, right, bottom]`) and labels as
   .AddOptionalArg("ratio",
       R"code(If true, bboxes returned values as expressed as ratio w.r.t. to the image width and height. Default: False)code",
       false)
+  .AddOptionalArg("size_threshold",
+      R"code(If width or height of a bounding box representing an instance of an object is under this value, object will be skipped during reading. It is represented as absolute value. Default value is 0.1)code",
+      0.1f,
+      false)
+  .AddOptionalArg("skip_empty",
+      R"code(If true, reader will skip samples with no object instances in them)code",
```

Comment:
```suggestion
      R"code(If true, reader will skip samples with no object instances in them. Default: False)code",
```

Reply:
That should automatically be populated in the docs, no?

Reply:
It will be. I think we need to go through operator description and remove this `Default` note

Reply:
I removed this unnecessary `Default` info for this op.